magento/adobe-stock-integration
# Add functional to remove Add Image button  in Standalone Media Gallery

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1103: Add Image button is available in Standalone Media Gallery 
2. ...

### Manual testing scenarios (*)

1.     From Admin go to Content - Media Media Gallery
2.     Select an image added from Adobe Stock
3.     Click on three dots near the image and select View details

**The Add Image button should not be present in Standalone Media Gallery**

**The Add image Button present in not standalone media gallery pages**
